docs/docs/02-Installation/04-kubernetes.md

```diff
@@ -0,0 +1,71 @@
+# Kubernetes
+
+### Requirements
+
+- A kubernetes cluster with at least a amd64 node.
```

Comment:
I'm curious, why do you need a single amd64? Hoarder (and all of its dependencies) work on arm as well.

Reply:
My cluster has a mix of amd64 and arm64 nodes. When I tried it, I was having issues with the chromium container. It could run but there was an error. I did not copy the error but I tried to run it on amd64 and everything worked.

So I have been a bit lazy not getting to the button of the issues.

But if everything is supposed to be arm64 compatible we can remove that restriction. I will update the PR.

Reply:
Done in [1b7ebf7](https://github.com/hoarder-app/hoarder/pull/324/commits/1b7ebf70a8272ec7f10838a72e9a1d4d860bbcf4)